align_system/algorithms/lib/persona/provider.py

```diff
@@ -102,32 +182,23 @@ def _choose_backstories_for_alignment_target(self, alignment_target: Optional[ty
 
         # Map the alignment target to a probe
         alignment_target_dict = alignment_target.to_dict() # type: ignore
-        probe_values = {
-            KDMA_TO_PROBE_MAPPING[k['kdma']]: k['value'] * 10
-            for k in alignment_target_dict.get('kdma_values', [])
-            if k['kdma'] in KDMA_TO_PROBE_MAPPING
-        }
 
         # Serialize the probe values into a repeatable string
-        probe_values_str = json.dumps(probe_values, sort_keys=True)
+        probe_values_str = yaml.dump(alignment_target_dict, sort_keys=True)
```

Comment:
I needed to switch this to YAML as the KDE alignment targets aren't easily serializable to JSON